server/test_route.py

```diff
@@ -38,5 +38,75 @@ def test_fetch_emotion(client):
 # TODO: sentiment testing
 # def test_fetch_sentiment(client):
 
-# TODO: summary testing
-# def test_fetch_summary(client):
+# Medium length input (400 words) - tests the basic API call works, and we get a response that 'summarises', i.e. shorter than the input.
+def test_fetch_summary_input_medium(client):
```

Comment:
Just wondering, what does `client` param do?

Reply:
`client` is a pytest parameter, I think it just initializes the app state or something? Was there when Stephen added it